packages/shared-ux/page/kibana_template/impl/src/page_template_inner.tsx

```diff
@@ -68,7 +68,8 @@ export const KibanaPageTemplateInner: FC<Props> = ({
       // the following props can be removed to allow the template to auto-handle
       // the fixed header and banner heights.
       offset={0}
-      minHeight={0}
+      minHeight={header ? 'calc(100vh - var(--euiFixedHeadersOffset, 0))' : 0}
+      grow={rest.grow ?? (header ? false : undefined)}
```

Comment:
```suggestion
      grow={header ? false : undefined}
```
very minor/optional nit - `rest.grow` will already override this `grow` prop due to the order of `...rest` below, so you can probably just leave it out here

Reply:
Applied